src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -130,9 +135,21 @@ async function initializeConnection(options: Required<BlazorOptions>, circuitHan
   }
 
   DotNet.attachDispatcher({
-    beginInvokeDotNetFromJS: (callId, assemblyName, methodIdentifier, dotNetObjectId, argsJson) => {
+    beginInvokeDotNetFromJS: (callId, assemblyName, methodIdentifier, dotNetObjectId, argsJson): void => {
       connection.send('BeginInvokeDotNetFromJS', callId ? callId.toString() : null, assemblyName, methodIdentifier, dotNetObjectId || 0, argsJson);
     },
+    endInvokeDotNetFromJS: (asyncHandle, succeeded, argsJson, replacer): void => {
```

Comment:
As per comments in https://github.com/aspnet/Extensions/pull/2043, I think this should be `endInvokeJSFromDotNet`, and should take a pre-serialized set of args instead of `argsJson` and `replacer`.

Reply:
I'm updating it to do that